revolv/payments/models.py

```diff
@@ -23,63 +36,92 @@ class PaymentInstrumentType(models.Model):
     objects = PaymentInstrumentTypeManager()
 
 
-class PaymentTransactionManager(models.Manager):
+class PaymentManager(models.Manager):
+    """
+        Simple manager for the Payment model.
+    """
+
+    def payments(self, user=None, project=None, queryset=None):
+        """
+        :return: Payments associated with this user
+        """
+        if queryset is None:
+            queryset = super(PaymentManager, self).get_queryset()
+        payments = queryset.filter(user=user).order_by('created_at')
+        return payments
+
+    def donations(self, user=None, project=None, queryset=None):
```

Comment:
Could you link me to the other manager functions you talk about?

If you were to DRY it out, would it be possible to have a private method called something like  `makeQuery` that each of these methods could just delegate to (call with right arguments)?